mantidproject/mantid
# Vesuvio - VesuvioCorrections and VesuvioCommands test should now pass on all platforms

- VesuvioCorrectionsTest Input parameters have been updated to improve the fit for the data
- Guard for older versions of boost has been added (was producing incorrect behaviour with random numbers #15937)

**To test:**
- Ensure the system tests pass on the build servers
- Run the system tests locally on a variety of OSs to ensure that the change works for various OSs

Fixes #15783 .

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
